[32mINFO    [0m utils.logger_util:request_util.py:29 准备发起GET请求，URL: http://101.42.36.43:8888/user/base-user/find-list
[32mINFO    [0m utils.logger_util:request_util.py:30 请求信息：{'headers': {'user_token': 'eyJhbGciOiJIUzI1NiJ9.eyJpZGVudGl0eSI6IlVTRVIiLCJ1c2VySWQiOjMsImlhdCI6MTc2MDg1NTE2OCwiZXhwIjoxNzYwOTQxNTY4fQ.b8coAM6oDyeYRdv1emuE2wpHTDt2KjVn0XT2Nz7uMvE'}, 'params': {'identity': 'ALL'}}
[32mINFO    [0m utils.logger_util:request_util.py:34 响应状态码：200
[32mINFO    [0m utils.logger_util:request_util.py:35 响应内容：{"code":500,"data":null,"msg":"数据库访问异常，请检查输入参数或联系管理员"}